core/snippets/snippets_insert_raw_text.py

```diff
@@ -16,9 +16,59 @@ class Stop:
     col: int
 
 
+stop_stack: list[Stop] | None = None
+
+
+def update_stop_information(stops: list[Stop]):
```

Comment:
I would move all these help/private functions below insert_snippet_raw_text

Reply:
I moved the helper functions below insert_snippet_raw_text to improve organization.